server/Mistakes.Journal.Api/Logic/Mistakes/Models/Repetition.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Mistakes.Journal.Api.Logic.Mistakes.Models
+{
+    public class Repetition
+    {
+        public Guid Id { get; }
+        public DateTime DateTime { get; set; }
```

Comment:
As I mentioned earlier, I think name `OccuredAt` is more readable, because `Datetime` does not say much about this attribute. I am open for other proposition :D